### PR TITLE
Remove Miller Column Browse template

### PR DESCRIPTION
**Pull request by chao-xian**
We have completed the AB Test for the New Browse template and want to switch from the Miller Column based template as the default.

[Preview on Heroku should show you only see the New Browse designs](https://collections-pr-2867.herokuapp.com/browse).

[Trello](https://trello.com/c/glBoTClk/1158-switch-off-new-browse-ab-test-and-show-new-browse-instead-of-miller-columns)

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

### CODE REVIEW

File: app/controllers/browse_controller.rb [hunk 1]
```diff
@@ -1,6 +1,4 @@
 class BrowseController < ApplicationController
-  enable_request_formats show: [:json]
```
Comment (hannako) [inline app/controllers/browse_controller.rb:2]: Why were we supporting json formats before, but not now. Not sure I understand how that relates to miller columns.

Comment (chao-xian) [reply app/controllers/browse_controller.rb:2]: The JSON was consumed by the Javascript to build up miller columns.

Comment (hannako) [reply app/controllers/browse_controller.rb:2]: Ah gotcha, thank you

**Comment by chao-xian** [comment]
Work from this PR is being incorporated into https://github.com/alphagov/collections/pull/2865

**Pull request closed without merging**